log.go

```diff
@@ -190,7 +190,7 @@ func formatCallerWithPathAndLine() (path string) {
 	return ""
 }
 
-var frameIgnorePattern = regexp.MustCompile(`github\.com/(S|s)irupsen/logrus`)
+var frameIgnorePattern = regexp.MustCompile(`github\.com/sirupsen/logrus`)
```

Comment:
@jakule So wouldn't it be more logical to change this regex to match `github.com/gravitational/logrus` and keep `frame.File` below? Unless I'm missing something.

Reply:
Yes, we can, but if we ever decide to change back to the upstream logrus we need to change it back. Current version work for any fork.